control/freqplot.py

```diff
@@ -429,12 +437,13 @@ def gen_zero_centered_series(val_min, val_max, period):
     else:
         return mags, phases, omegas
 
+
 #
 # Nyquist plot
 #
 
-def nyquist_plot(syslist, omega=None, Plot=True,
-                 labelFreq=0, arrowhead_length=0.1, arrowhead_width=0.1, 
+def nyquist_plot(syslist, omega=None, plot=True, labelFreq=0,
```

Comment:
change to `label_freq` for the same reason as the change to `plot` and `print_gain`?